CalendarApp/GoogleCalendarService/BookingService.cs

```diff
@@ -4,23 +4,34 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using GoogleCalendarService.Manager;
 
 namespace GoogleCalendarService
 {
     public class BookingService
     {
-        private CalendarService _service;
-        private ILog _log;
+        private readonly CalendarService _service;
+        private readonly ILog _log;
+        public EventManager EventManager;
+        public string TimeZone;
 
         public BookingService(CalendarService service, ILog log)
         {
             _service = service;
             _log = log;
+            EventManager = new EventManager(this);
```

Comment:
Such initialization can lead to bit coupling between these classes. Could it be avoided using IoC container?